php_phongo.c

```diff
@@ -380,6 +380,7 @@ void phongo_session_init(zval* return_value, mongoc_client_session_t* client_ses
 
 	session                 = Z_SESSION_OBJ_P(return_value);
 	session->client_session = client_session;
+	session->client         = mongoc_client_session_get_client(client_session);
```

Comment:
A `client` field is needed on the cursor struct since libmongoc's public API provides no method to access a cursor's underlying client. We obviously don't need that here, but creating the field allows the macros below to universally operate on a Manager, Cursor, or Session object.

If you don't see a problem with this, I'll refactor other calls to `mongoc_client_session_get_client()` in `Session.c` to simply refer to this new field in a separate commit.

Reply:
> I'll refactor other calls to `mongoc_client_session_get_client()` in `Session.c`

That sounds like a good plan 👍

Reply:
This is now done.